oam/install_scripts/columnstore-post-install.in

```diff
@@ -315,9 +315,14 @@ systemctl cat mariadb-columnstore.service > /dev/null 2>&1
 if [ $? -eq 0 ] && [ $(running_systemd) -eq 0 ]; then
     mkdir -p /var/lib/columnstore/storagemanager
     chown -R mysql:mysql /var/lib/columnstore/storagemanager
+    IFLAG=/var/lib/columnstore/storagemanager/storagemanager-lock
```

Comment:
I believe the operations run by different nodes could overlap b/c file create isn't an atomic operation however it should be good for most cases.

Reply:
Good point, this does not need to be atomic as inside mariadb-columnstore-start.sh we have an atomic flock to prevent nodes running simultaneously from stepping on each other. this is purely meant for nodes added after an install/initialization of the first node